examples/c/ex_file_system.c

```diff
@@ -31,6 +31,12 @@
 #include <errno.h>
 #include <stdlib.h>
 #include <string.h>
+#ifndef _WIN32
+#include <unistd.h>
```

Comment:
Added so that it uses the same logic as the other example files.

I don't think its needed in this case but wanted to keep the examples looking similar.

Can remove if need be.